src/utilities/helpers.spec.ts

```diff
@@ -89,4 +89,17 @@ describe('helpers', () => {
       expect(request).toHaveBeenNthCalledWith(2, { offset: 500, size: 500 })
     })
   })
+
+  describe('camelCaseToWords', () => {
+    test.each([
+      ['test', 'test'],
+      ['MeshOPAPolicy', 'Mesh OPA Policy'],
+      ['MeshOPA', 'Mesh OPA'],
+      ['MeshCircuitBreaker', 'Mesh Circuit Breaker'],
+      ['Mesh Circuit Breaker', 'Mesh Circuit Breaker'],
+      ['meshCircuitBreaker', 'mesh Circuit Breaker'],
```

Comment:
Didn’t we want to stop relying on `singularDisplayName` and `pluralDisplayName`?

In any case, the place where the utility `camelCaseToWords` is used also displays data from other fields, not just policy types, so this logic would still be needed even if we looked up the display name in the policy type data.

Reply:
If we switch to not use `singularDisplayName` let's just display policies by their name without trying to play with casing.

Reply:
This isn’t really about casing but about splitting camel-case strings into space-delimited words for the purposes of displaying them here:

![image](https://user-images.githubusercontent.com/5774638/214534442-aef5d30a-0574-4556-9fc9-0f16d6e5c5d1.png)

This is how `responsesSent` becomes `Responses Sent`, etc.

On top of that, this component shows some content (including policy types) in all uppercase letters.

Besides transforming all letters to uppercase in this particular view, we already don’t ever touch the casing of policy type names.

Reply:
Could we just use a map of `responsesSent: "Responses Sent"`? And then the policy names are just whatever comes out of the API (as @lahabana says)

Reply:
I feel like we’re rather getting away from the issue of certain policy types (e.g. MeshOPAPolicy) being incorrectly displayed. We can certainly change how we display text here, but this is out of scope of this issue.

Reply:
It sounds like we should use `singularDisplayName` or even just `name` (the thing in `name` is probably what I would google for, i.e. without the space - but thats just me), and then for anything that isn't something that has a name like `responsesSent` we could just use a map. This then solves the isse of the policy names being incorrectly displayed and also solves:

> This is how responsesSent becomes Responses Sent, etc.

Apologies if I've misunderstood anything here, I'm not super familiar with the API endpoint for this, and just going by what's been said in this PR.

Reply:
I’ve now removed the usage of `camelCaseToWords` entirely. This also addresses the issue, ensures policy type names appear as their regular resource names (e.g. "CircuitBreaker"), but fields like `responsesSent` now appear as camel-cased. I’m not addressing this now because it would increase the scope further.